docs/tutorial/04-capabilities.mdx

```diff
@@ -0,0 +1,418 @@
+(Covered in the next section)
+
+Lastly, we call the `hello()` method with our borrowed reference:
+
+```cadence
+// Call the hello function using the reference to the HelloAsset resource
+log(helloReference.hello())
+```
+
+At the end of the transaction execution, the `helloReference` value is lost,
+but that is ok because while it references a resource, it isn't the actual resource itself, so it is ok to lose it.
+
+In the next section, we look at how capabilities can expand the access a script has to an account.
+
+### Executing Scripts
+
+---
+
+A script is a very simple transaction type in Cadence that cannot perform
+any writes to the blockchain and can only read the state of an account or contract.
+It runs without permission from any account, so scripts need capabilities to access account storage.
+
+To execute a script, write a function called `pub fun main()`.
+You can click the execute script button to run the script.
+The result of the script will be printed to the console output.
+
+<Callout type="info">
+
+Open the file `Script1.cdc`.
+
+<br />
+
+`Script1.cdc` should look like the following:
+
+</Callout>
+
+```cadence:title=Script1.cdc
+import HelloWorld from 0x01
+
+pub fun main() {
+
+    // Cadence code can get an account's public account object
+    // by using the getAccount() built-in function.
+    let helloAccount = getAccount(0x01)
+
+    // Get the public capability from the public path of the owner's account
+    let helloCapability = helloAccount.getCapability<&HelloWorld.HelloAsset>(/public/HelloAssetTutorial)
+
+    // borrow a reference for the capability
+    let helloReference = helloCapability.borrow()
+        ?? panic("Could not borrow a reference to the hello capability")
+
+    // The log built-in function logs its argument to stdout.
+    //
+    // Here we are using optional chaining to call the "hello"
+    // method on the HelloAsset resource that is referenced
+    // in the published area of the account.
+    log(helloReference.hello())
+}
+```
+
+Here's what this script does:
+1. It fetches the `PublicAccount` object with `getAccount` and assigns it to the variable `helloAccount`
+2. Uses the `getCapability` method to get the capability from the `Create Link` transaction
+3. Borrows a reference for the capability using the `borrow` method and assigns it to `helloReference`
+4. Logs the result of the `hello()` function from `helloReference` to the console.
+
+```cadence
+let helloAccount = getAccount(0x01)
+```
+
+The `PublicAccount` object is available to anyone in the network for every account,
+but only has access to a small subset of functions that can be read from the `/public/` domain in an account.
+Most of the [account storage API](https://docs.onflow.org/cadence/language/accounts/#account-storage-api) is not available on the `PublicAccount` object, and only signed transactions can access the `AuthAccount` object.
```

Comment:
Might be worth calling out here that with the Secure Cadence changes, it is actually possible to access `AuthAccount` objects with the `getAuthAccount` function, which only exists in scripts because all changes are discarded after execution.